app/models/member.rb

```diff
@@ -32,6 +32,7 @@ class Member < ActiveRecord::Base
   scope :confirmed, -> { where('confirmed_at IS NOT NULL') }
   scope :located, -> { where("location <> '' and latitude IS NOT NULL and longitude IS NOT NULL") }
   scope :recently_signed_in, -> { reorder('updated_at DESC') }
+  scope :recently_joined, -> { reorder("created_at desc") }
```

Comment:
You may be right but technically a user could be considered as joined as soon as they registered.